solutions/arithmetic/find-fn.js

```diff
@@ -0,0 +1,23 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+  idx++;
+  return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+// Use readLine() for taking input, it will read one line of from the input  and is stored in string format
+
+let num = BigInt(readLine());
```

Comment:
I changed the test cases to use smaller numbers. Let us not go for `BigInt`. In the initial days it will be too much for the students.